src/nxdoc/web-ui/web-ui-tutorials/web-ui-configuration-properties.md

```diff
@@ -112,6 +112,13 @@ An example of its usage can be found on the [Nuxeo Date Picker](https://github.c
 </component>
 ```
 
+### Timezone
+
+By default, Web UI displays dates according to the timezone detected by the user's web browser.
+However, by configuring the `timezone` setting, it's possible to specify the name of a specific timezone (according to the [IANA tz database](https://www.iana.org/time-zones)) to be used globally instead.
+For the moment, the only valid value for this setting is `Etc/UTC`, which assumes that the time specified by the user is UTC.  
+
+
```

Comment:
I think we should provide an XML sample of how to configure this for Web UI, like we do on the other examples. Also, could you please confirm with @manonlumeau if we need to backport this to any branch for LTS 2021?

Reply:
I haven't created the `2021` branch yet, to avoid any backports for now. I'll create it at code freeze.